### PR TITLE
feat(chart): create chart folder

### PR DESCRIPTION
**Pull request by ArthusLiang**
## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Documentation update
- [X] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [X] Build related changes
- [X] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: chart/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+bundles/
+es/
+lib/
+!src/lib
```
Comment (mathsusan) [inline chart/.gitignore]: !src/lib doesn't seem like a good idea ..

Comment (ArthusLiang) [reply chart/.gitignore]: I want to enable root/src/lib here. But I do want to enble root/lib. 
root/src/lib stores the source code
root/lib.      stores the built version
Do you know any better idea?

Comment (mathsusan) [reply chart/.gitignore]: did you look at the documents https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository#_ignoring ?

Comment (ArthusLiang) [reply chart/.gitignore]: Thanks for this. Now I use the config in the Momentum Root

**Pull request merged**